docs/internal-documentation/templates/specifications/tests/TestCaseIdentifierSpecification.md

```diff
@@ -0,0 +1,32 @@
+# Test Case Identifier Specification
+
+All test cases belong to one specific test level, and it gets its name from that
+test level's name. The following test levels are defined and available:
+
+* Address
+* Basic
+* Connectivity
+* Consistency
+* DNSSEC
+* Delegation
+* Nameserver
+* Syntax
+* Zone
+
+The test case identifier has the format: \<Test Level name> + \<Index>
```

Comment:
Maybe use curly braces instead of angle brackets? Just avoid escaping in markdown.

Reply:
That is a good suggestion. I'll update.